asciidoc-release-notes/adoc_postprocess.xsl

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Shipping that as part of the release notes is not ideal (we have a separate but likely older version of this stylesheet as part of DAPS). I guess for the time being, it makes sense to include it here, at least this way, it's easy to update.